### ISSUE TITLE
CUDNN_STATUS_INTERNAL_ERROR

### ISSUE THREAD

**Issue by andyhoeung** [Purfview/whisper-standalone-win#334]
Error I get now with r194.1.

![image](https://github.com/user-attachments/assets/e165095b-5007-493a-8699-830f951f2816)

**Comment by Purfview** [comment]
Could you check if you get same error in `r193.1` version?

**Comment by Purfview** [comment]
Why you set `--device cpu`?

**Comment by andyhoeung** [comment]
> Could you check if you get same error in `r193.1` version?

<img width="875" alt="image" src="https://github.com/user-attachments/assets/f0b24add-d6cd-4302-8d60-bfdafbab3d76">

> Why you set `--device cpu`?

Without explicitly saying 'cpu', it defaults to CUDA and gives the error above. (AMD GPU user) With r193.1, it works fine when I enter that argument, so I also did that for r194.1. Maybe it's a conflict with Zluda stuff I have installed for Stable Diffusion, seeing as it's mentioned in the screenshot.
